src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -893,6 +893,25 @@ try
         settings->_hash = _calculateHash(settingsString, lastWriteTime);
     }
 
+    // GH#13936: We're interested in how many users opt out of useAtlasEngine,
+    // indicating major issues that would require us to disable it by default again.
+    {
+        size_t enabled[2]{};
+        for (const auto& profile : settings->_activeProfiles)
+        {
+            enabled[profile.UseAtlasEngine()]++;
+        }
+
+        TraceLoggingWrite(
+            g_hSettingsModelProvider,
+            "AtlasEngine_Usage",
+            TraceLoggingDescription("Event emitted upon settings load, containing the number of profiles opted-in/out of useAtlasEngine"),
+            TraceLoggingUIntPtr(enabled[0], "UseAtlasEngineDisabled", "Number of profiles for which AtlasEngine is disabled"),
+            TraceLoggingUIntPtr(enabled[1], "UseAtlasEngineEnabled", "Number of profiles for which AtlasEngine is enabled"),

```

Comment:
Do we maybe want to break this down further?
* Users that don't have the setting at all (`!HasUseAtlasEngineDisabled()`)
* Users that explicitly opted in
* Users that explicitly opted out

and maybe a total number of profiles too? Trying to figure out how many folks actually just set this in `profiles.defaults` might complicate this...

Reply:
Ah right, we can infer that the user opted out if Disabled > 0. The version **is** part of the data we automatically get.

Reply:
Oh so I guess (Num Preview users)-(preview users with it disabled)=(preview users with it enabled, either by default or explicitly). Okay, that makes sense.